### DISCUSSION TITLE
I'm working on a dinosaur

### DISCUSSION THREAD

**Discussion by JAE1985** [DSheirer/sdrtrunk#1504] [Q&A] [answered]
I'm working from an old laptop and I was wondering if there's a version of this that will work with a windows x64 processor?  If not then would anyone know of a different program that would work and where to find it?  Any help would be greatly appreciated.

**Comment by wmg819** [comment, marked as answer]
SDRTrunk runs on windows X64 systems but a little more information such as your computers processor speed, number of cores and the memory available would help in getting a definite answer. I have the current SDRTrunk version running on a system with Windows 7 X64 with an i5-2300 cpu  and 4 cores with no problems. Any windows X64 system running on 4 cores with 8 GB of ram should work.

**Comment by DSheirer** [comment]
Please post end-user questions like these to any of the support forum options listed in the wiki page.

The github issues are for reporting application bugs and requesting feature enhancements.

**Reply by JAE1985** [reply]
I apologize I was not aware.
